### PR TITLE
clarify and fix problems in paddle on aws k8s (create cluster part)

### PR DESCRIPTION
**Pull request by helinwang**
This is the second pass. @cxwangyi @tizhou86 @backyes

### CODE REVIEW

File: doc/howto/usage/k8s/k8s_aws_en.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
 
 ## Create AWS Account and IAM Account
 
-AWS account allow us to manage AWS from Web Console. Amazon AMI enable us to manage AWS from command line interface.
+AWS account allow us to manage AWS from Web Console. Amazon IAM enable us to manage AWS from command line interface.
```
Comment (wangkuiyi) [inline doc/howto/usage/k8s/k8s_aws_en.md]: 是不是这样能更清楚地表达AWS account和IAM users之间的关系？

> Under each AWS account, we can create multiple [IAM](http://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) users. This allows us to grant some privileges to each IAM user and to create/operate AWS clusters as an IAM user.

File: doc/howto/usage/k8s/k8s_aws_en.md [hunk 2]
```diff
@@ -194,29 +197,37 @@ paste into following inline policies:
...
+The A record of that DNS name needs to be point to the cluster ip address.
+
+We will need to use DNS name later in tutorial. If you don't already own one, you can choose any DNS name (e.g., `paddle`) and modify `/etc/hosts` to associate cluster ip with that DNS name.
 
 #### S3 bucket
 
 You need to create an S3 bucket before startup the Kubernetes cluster.
 
-command (need to have a global unique name):
+There are some bug in aws cli in creating S3 bucket, so let's use [web console](https://console.aws.amazon.com/s3/home?region=us-west-1).
```
Comment (wangkuiyi) [inline doc/howto/usage/k8s/k8s_aws_en.md]: bug ==> bugs
web ==> Web

Comment (wangkuiyi) [inline doc/howto/usage/k8s/k8s_aws_en.md]: use web console ==> use the Web console

File: doc/howto/usage/k8s/k8s_aws_en.md [hunk 3]
```diff
@@ -2,18 +2,18 @@
 
 ## Create AWS Account and IAM Account
 
-AWS account allow us to manage AWS from Web Console. Amazon AMI enable us to manage AWS from command line interface.
+AWS account allow us to manage AWS from Web Console. Amazon IAM enable us to manage AWS from command line interface.
 
-We need to create an AMI user with sufficient privilege to create kubernetes cluster on AWS.
+We need to create an IAM user with sufficient privilege to create kubernetes cluster on AWS.
```
Comment (wangkuiyi) [inline doc/howto/usage/k8s/k8s_aws_en.md]: 我感觉这句话可以删了，因为 L16 列出来了需要的privileges了。

File: doc/howto/usage/k8s/k8s_aws_en.md [hunk 4]
```diff
@@ -59,7 +51,7 @@ gpg2 --fingerprint FC8A365E
 ```
 The correct key fingerprint is `18AD 5014 C99E F7E3 BA5F 6CE9 50BD D3E0 FC8A 365E`
 
-Go to the [releases](https://github.com/coreos/kube-aws/releases) and download the latest release tarball and detached signature (.sig) for your architecture.
+Go to the [releases](https://github.com/coreos/kube-aws/releases) and download release tarball (this tutorial is using v0.9.1) and detached signature (.sig) for your architecture.
```
Comment (wangkuiyi) [inline doc/howto/usage/k8s/k8s_aws_en.md]: > We can download kube-aws from its [release page](https://github.com/coreos/kube-aws/releases).  In this tutorial, we use version 0.9.1.

File: doc/howto/usage/k8s/k8s_aws_en.md (hunk 1 above)
Comment (helinwang) [inline doc/howto/usage/k8s/k8s_aws_en.md]: Yes, great! Done.

File: doc/howto/usage/k8s/k8s_aws_en.md (hunk 2 above)
Comment (helinwang) [inline doc/howto/usage/k8s/k8s_aws_en.md]: Done.

Comment (helinwang) [inline doc/howto/usage/k8s/k8s_aws_en.md]: Done.

File: doc/howto/usage/k8s/k8s_aws_en.md (hunk 3 above)
Comment (helinwang) [inline doc/howto/usage/k8s/k8s_aws_en.md]: Done.

File: doc/howto/usage/k8s/k8s_aws_en.md (hunk 4 above)
Comment (helinwang) [inline doc/howto/usage/k8s/k8s_aws_en.md]: Done.

**Pull request merged**